lib/WebDriverWindow.php

```diff
@@ -79,10 +84,28 @@ public function getSize()
      */
     public function maximize()
     {
-        $this->executor->execute(
-            DriverCommand::MAXIMIZE_WINDOW,
-            [':windowHandle' => 'current']
-        );
+        if ($this->isW3cCompliant) {
+            $this->executor->execute(DriverCommand::MAXIMIZE_WINDOW, []);
+        } else {
+            $this->executor->execute(
+                DriverCommand::MAXIMIZE_WINDOW,
+                [':windowHandle' => 'current']
+            );
+        }
+
+        return $this;
+    }
+
+    /**
+     * Makes the current window full screen.
+     *
+     * @return WebDriverWindow The instance.
+     */
+    public function fullscreen()
+    {
+        if ($this->isW3cCompliant) {
```

Comment:
The fullscreen was not part of JsonWire (but may work eg. in Chrome), so I think we should either:
a) execute the fullscreen command regaredless the protocol (and let i fail by exception on the remote end)
b) throw exception if not in w3c mode (like we do in `minimize()`)

But silently doing nothing is IMO not the best approach.